pages/contribute.md

```diff
@@ -6,44 +6,59 @@ main_nav_id: contribute
 
 # Contributing to Kotlin
 
-Kotlin is an Apache 2 OSS Project. The source code, tooling, documentation and even this web site is maintained on [GitHub](https://github.com/jetbrains/kotlin). While the majority 
-of the team works at JetBrains, there have been nearly a hundred external contributors to the Kotlin Project and we are always looking for more people. 
+Kotlin is an Apache 2 OSS Project. The source code, tooling, documentation and even this web site is maintained on [GitHub](https://github.com/jetbrains/kotlin). 
+While Kotlin is mostly developed by JetBrains, there are hundreds of external contributors to the Kotlin project&mdash;[become one of them](https://kotlinlang.org/community). 
 
-## How you can contribute
+## Participate in Early Access Preview
 
-### Contributing to the code
+You can help us improve Kotlin by participating in Kotlin Early Access Preview (EAP) and providing us with your valuable feedback.
```

Comment:
Am I correct, you will add here instruction on how to get access to EAP later? @katia-energizer

Reply:
That's right. I will add a link once we have articles published on the website.